packages/commonwealth/client/scripts/navigation/CustomDomainRoutes.tsx

```diff
@@ -307,13 +321,51 @@ const CustomDomainRoutes = () => {
     // CONTRACTS END
 
     // ADMIN
-    <Route
-      key="/manage"
-      path="/manage"
-      element={withLayout(ManageCommunityPage, {
-        scoped: true,
-      })}
-    />,
+    ...(featureFlags.newAdminOnboardingEnabled
+      ? [
+          <Route
+            key="/community-profile"
+            path="/community-profile"
+            element={withLayout(CommunityProfile, {
+              scoped: true,
+            })}
+          />,
+          <Route
+            key="/community-integrations"
+            path="/community-integrations"
+            element={withLayout(CommunityIntegrations, {
+              scoped: true,
+            })}
+          />,
+          <Route
+            key="/community-topics"
+            path="/community-topics"
+            element={withLayout(CommunityTopics, {
+              scoped: true,
+            })}
+          />,
+          <Route
+            key="/community-moderators"
+            path="/community-moderators"
+            element={withLayout(CommunityAdminAndModerators, {
+              scoped: true,
+            })}
+          />,
+        ]
+      : [
+          <Route
+            key="/:scope/manage"
+            path="/:scope/manage"
```

Comment:
Actually, that was in `master` branch, so I didn't remove that (Not sure if removing it will break anything).

https://github.com/hicommonwealth/commonwealth/blob/7e1a8187b99d06e71294b32d1c45f631a92878a4/packages/commonwealth/client/scripts/navigation/CustomDomainRoutes.tsx#L566-L570

Reply:
oh it is redirect, so I think thats fine!